ee/tabby-db/migrations/0042_page-section.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE pages(
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+
+    -- Whether the page is draft
+    is_draft BOOLEAN NOT NULL,
+
+    -- The user who created the page
+    user_id INTEGER NOT NULL,
+
+    -- Array of relevant questions, in format of `String`
+    relevant_questions BLOB,
+
+    created_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+    updated_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+
+    FOREIGN KEY(user_id) REFERENCES users(id) ON DELETE CASCADE
+);
+
+CREATE TABLE page_sections(
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    page_id INTEGER NOT NULL,
+
+    order_index INTEGER NOT NULL,
```

Comment:
```suggestion
    index INTEGER NOT NULL,
```

Reply:
both `order` and `index` is keyword in db, it should be not recommended to use them?